plugins/com.google.cloud.tools.eclipse.preferences/plugin.xml

```diff
@@ -14,18 +14,12 @@
       </page>
    </extension>
    <extension
```

Comment:
Oh right you were fetching the value directly via ConfigurationScope.  It makes more sense if we were using the IPreferenceService, which will look up a set of scopes (default: instance, configuration, then default scope).